CriticalMass/ChatInputView.swift

```diff
@@ -139,3 +91,17 @@ private extension String {
         !self.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty
     }
 }
+
+private extension UIButton {
+    func setBackgroundColor(color: UIColor, forState: UIControl.State) {
```

Comment:
I don’t really like that solution. If there is no other way we should use the UIGraphicsImageRenderer api. 
Maybe a similar to approach to CustomButton.swift might be an optio.

Reply:
Was the simplest one without subclassing UIButton. Would you draw a shape?

Reply:
I would invert the shape of the pdf and just use the default behavior that a UIButton offers when it’s marked as disabled

Reply:
maybe you can post some example code here

Reply:
![Group](https://user-images.githubusercontent.com/7677738/68467310-421db280-0216-11ea-9db7-a3fc314ee4c7.png)
 basically I would suggest using that shape as the button image (the arrow should be transparent). That would solve the issue around corner radius for changing frames + give us the default disabled behaviour without any custom logic

Reply:
Ok. Then I think we need an asset update. I think the current approach fits in the theming control. Now I moved the cornerRadius in the layoutSubviews

Reply:
Giving it a second thought I don’t think we should base the button implementation on the asset. I’m in favor of having a iconbutton class maybe. The button will not grow in size increasing dynamic type anyways